### ISSUE TITLE
MONDO:0006251 and MONDO:0016039 [Merge]

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#4093]
**Mondo term (ID and Label)**
MONDO:0006251 
inclusion body fibromatosis

**Reason for deprecation**
HPO and NCI say these terms are synonymous https://hpo.jax.org/app/browse/term/HP:0025197 
https://ncithesaurus.nci.nih.gov/ncitbrowser/pages/home.jsf;jsessionid=0574F720B1BE42E0A84DF866EF29A04F

**Term to be merged with**
MONDO:0016039 infantile digital fibromatosis
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by nicolevasilevsky** [comment]
note to self:
sh run.sh owltools --use-catalog mondo-edit.obo  --obsolete-replace MONDO:0006251 MONDO:0016039 -o -f obo mondo-edit.obo
